system-reinstall-bootc/src/podman.rs

```diff
@@ -16,7 +16,8 @@ pub(crate) fn command(image: &str, root_key: &Option<UserKeys>) -> Command {
         // The container needs to access the host's PID namespace to mount host directories
         "--pid=host",
         // Set the UID/GID to root overwriting any possible USER directive in the Containerfile
-        "--user root:root",
+        "--user",
+        "root:root",
```

Comment:
you can use `--user=root:root` if you want (I forgot it)

Reply:
I cannot approve or lgtm in this repo, but LGTM :)

Reply:
> you can use `--user=root:root` if you want (I forgot it)

Yeah that's definitely better for consistency with the other options used